dev/com.ibm.ws.security.oauth/src/com/ibm/oauth/core/internal/oauth20/responsetype/impl/OAuth20ResponseTypeHandlerCodeImpl.java

```diff
@@ -112,6 +116,55 @@ public void validateRequestResponseType(AttributeList attributeList, JsonArray r
             result = new ArrayList<OAuth20Token>();
             result.add(code);
 
+            SubjectHelper subjectHelper = new SubjectHelper();
```

Comment:
If we have a chance to come back to this, this new section would probably be better suited in a separate method if at all possible to try to keep this method from growing too big.